site/pages/api/create-social-media-post.api.ts

```diff
@@ -95,15 +95,26 @@ const createSocialMediaPost = async (req: NextApiRequest, res: NextApiResponse):
             await putItem(process.env.IMAGE_BUCKET_NAME || "", validatedBody.data.image.key, imageContents);
         }
 
-        await upsertSocialMediaPost(
-            validatedBody.data.status === SocialMediaPostStatus.rejected
-                ? validatedBody.data
-                : { ...validatedBody.data, status: SocialMediaPostStatus.pending },
-            session.orgId,
-            session.isOrgStaff,
-            false,
-            template === "true",
-        );
+        const socialMediaToUpsert =
+            template === "true" || validatedBody.data.accountType === "Twitter"
+                ? {
+                      ...validatedBody.data,
+                      publishTime: "",
```

Comment:
Is this required?

Reply:
yes for current templates these should not have date/time

Reply:
as the field is optional - removing this and next line didnt seem to cause any issues in terms of testing the functionality

Reply:
but in the data when it become a disruption it will have those pre populated dates that were entered before

Reply:
added a comment

Reply:
In the social media page if you enter time and not date and then click save - you are redirected to the same page with error but the time is removed from the input box after redirection